FHIR-Converter/README.md

```diff
@@ -13,69 +13,98 @@ This extension provides an interactive editing and verification experience to cr
+
+You can use the command `FHIR Converter: Login to Azure Container Registry (Ctrl + I)`:
+- First, you need to enter the private ACR `<registry-name>.azurecr.io`, and then the login command will run in the terminal using oras tool;
+- Second, you need to input the username and password interactively. We will not store any information about your username and password.
+
+If you have installed Azure CLI or Docker, you can refer to this [documentation](https://github.com/microsoft/FHIR-Converter/blob/main/docs/TemplateManagementCLI.md#authentication) to log in by its ways. 
+
+After that, you can run the command `FHIR Converter: Logout registry (Ctrl + O)` to logout from ACR. In this command, you only need to enter the private registry.
+
+![login-logout](assets/login-logout.gif)
+
+### 2. Pull templates
+
+If you don’t have templates yet, you can use the command `FHIR Converter: Pull Microsoft templates (Ctrl + D)` to pull the default templates publicly published by Microsoft.
+
+If you want to use the private registry, you can use the command `FHIR Converter: Pull templates (Ctrl + L)` to pull templates:
+- Firstly, login to ACR;
+- Secondly, enter your image reference `<registry>/<image-name>:<tag>` (image name should be lowcase); 
+- Thirdly, select the output folder to store the templates. If the output folder is not empty, a prompt will pop up. If you choose to force overwrite, the files with the same name will be overwritten in the output folder.
+
+![pull-templates](assets/pull-templates.gif)
+
+### 2. Pull sample data
+
+If you don’t have data yet, you can use the command `FHIR Converter: Pull sample data (Ctrl + T)` to pull the sample data publicly published by Microsoft. In this command, you only need to select the output folder to store the data. As with the pull command, if the output folder is not empty, a prompt will pop up. If you choose to force overwrite, the files with the same name will be overwritten in the output folder.
+
+![pull-sample-data](assets/pull-sample-data.gif)
+
+### 3. Create a converter workspace
 
 If the converter workspace has not been created, you need to create a new converter workspace by triggering the command `FHIR Converter: Create a converter workspace (Ctrl + W)` .
 
-After triggering the command, some actions need to be completed for creating a converter workspace as follows:
+After the command is triggered, some actions need to be completed for creating a converter workspace as follows:
 
-- First, select a root template folder.
+- Firstly, select a root template folder;
 
-- Second, select a root data folder.
+- Secondly, select a data folder;
   
-- Third, select a workspace folder to store the workspace configuration file, and then input the file name for the workspace (*.fhir-converter.code-workspace).
+- Thirdly, select a workspace folder to store the workspace configuration file, and then input the file name for the workspace (*.fhir-converter.code-workspace).
```

Comment:
Select a workspace file name. The file name extension fhir-converter.code-workspace will be added automatically.